src/webgpu/api/validation/createRenderPipeline.spec.ts

```diff
@@ -248,6 +249,56 @@ g.test('color_formats_must_be_renderable')
     t.doCreateRenderPipelineTest(isAsync, info.renderable && info.color, descriptor);
   });
 
+g.test('formats_for_depth_stencil_state_must_be_depth_stencil_formats')
+  .params(u => u.combine('isAsync', [false, true]).combine('format', kTextureFormats))
+  .fn(async t => {
+    const { isAsync, format } = t.params;
+    const info = kTextureFormatInfo[format];
+    await t.selectDeviceOrSkipTestCase(info.feature);
+
+    const descriptor = t.getDescriptor({ depthStencil: { format } });
+
+    t.doCreateRenderPipelineTest(isAsync, info.depth || info.stencil, descriptor);
+  });
+
+g.test('format_aspect_requirement_for_depth_stencil_test')
```

Comment:
Can also be shorter like `depth_stencil_attachment,aspect`

Reply:
Done. And I slightly changed the names because I separated tests further.